### PR TITLE
port to sinatra

### PR DESCRIPTION
**Pull request by orta**
Migrates the pod_pages branch to pure sinatra

### CODE REVIEW

File: views/layout.slim [hunk 1]
```diff
@@ -2,12 +2,13 @@ doctype html
...
     title = "CocoaPods.org"
-        
+
     == shared_partial("header_meta")
     == shared_partial("favicons")
-    
+
     script src="/javascripts/application.js"
     link href="/stylesheets/app.css" media="screen" rel="stylesheet" type="text/css"
+    link rel="search" type="application/opensearchdescription+xml" href="http://cocoapods.org/opensearch" title="CocoaPods"
```
Comment (kylef) [inline views/layout.slim]: This is a 404, it has an extension of .xml?

```
get '/opensearch.xml' do
```

Comment (orta) [inline views/layout.slim]: oh yeah, good point!

Comment (orta) [inline views/layout.slim]: This was already fixed in a later commit👍

File: views/summer-of-code-2015.slim [hunk 2]
```diff
@@ -24,20 +24,20 @@ css:
...
 
-          ul#projects
-            - for subproject in project.projects
-              li
-                p = "#{subproject.repos} - #{subproject.title}"
-                a
-                  a href="#{subproject.issue_url}" = subproject.issue_url.split("github.com/")[-1].gsub("/issues/", "#")
-                p = subproject.description
-                hr
+          / ul#projects
```
Comment (kylef) [inline views/summer-of-code-2015.slim]: This doesn't seem to be related to the main purpose of this PR (Sinatra).

What's the intention here?

Comment (orta) [inline views/summer-of-code-2015.slim]: was having nil issues, think its :key vs "key'
